Tools/clinic/clinic.py

```diff
@@ -4901,8 +4959,13 @@ def state_parameter(self, line: str) -> None:
             self.parameter_continuation = line[:-1]
             return
 
+        line = line.lstrip()
+        if match := self.deprecate_posonly_re.match(line):
```

Comment:
What is a reason of using `:=` here? It saves a line of code, but makes it less clear.
```suggestion
        match = self.deprecate_posonly_re.match(line)
        if match:
```

Reply:
I have no style preference here; I can make it two lines, if you think that is clearer.

Reply:
Yes, I think that it is clearer. With `:=` I need to jump back when read this line.